app/graphql/types/record_type.rb

```diff
@@ -126,6 +126,19 @@ class LocationType < Types::BaseObject
       field :kind, String
       field :value, String
     end
+
+    class HighlightType < Types::BaseObject
+      field :kind, String
```

Comment:
I wonder if `:field` or `:matched_field` would be more useful here than `:kind`?